docs/developer-guide/etendo-classic/how-to-guides/How-to-Create-a-Chart-of-Accounts-Module.md

```diff
@@ -0,0 +1,106 @@
+
+!!!important
+    A Chart of Accounts Module for Etendo can only contain one chart of accounts. So, if you are working in a localization project for a country where several chart of accounts can be used, you must create independent modules including each chart of accounts CSV file. It will be up to the user to install and apply the chart of accounts module that fits particular needs.
+
+## Creating the CSV file
+
+The first step for creating a chart of accounts module is to write the CSV file with the list of accounts we want to include. All the information can be found in [How to Create Accounts Files](How-to-Create-Accounts-Files.md).
+
+## Testing the CSV file
+
+Before creating the Chart of Accounts module, we should first check that the CSV file has no structural or logical errors. This can be easily done in two steps:
+
+1. The first one is using a testing spreadsheet provided by Etendo that will detect any structural error, like missing search keys, wrong values in Element Level, Parent_Value or Show Value Condition columns, etc.
+
+2. The second check can be directly executed through the ERP. It just consists on importing our CSV file when running the Initial Client Setup process.
+Finally, there is also recommended to create some accounting entries for every transactional document and ensure the accounting report information is OK.
+
+!!!info
+    Please visit [Chart of Accounts Testing]() to learn more about these steps.
+
+## Creating the Chart of Accounts module
+
+The remaining step is clearly the easiest one, which is just including the CSV file into our Chart of Accounts module.
+
+A Chart of Accounts module is like any other module, but it has just the following considerations:
+
+- The CSV file must be included into the `referencedata/accounts/` directory of our module.
+
+- The file must be renamed to COA.csv. So the directory structure for our module should look like:
+
+```java
+ <module java package name>
+   ├── referencedata 
+   │   └── accounts 
+   │       └── COA.csv 
+   └── src-db 
+       └──  database 
+           └── sourcedata 
+             ├── AD_DATASET.xml 
+             ├── AD_MENU.xml 
+             ├── AD_MODULE_DBPREFIX.xml 
+             ├── AD_MODULE_DEPENDENCY.xml 
+             ├── AD_MODULE.xml 
+             ├── AD_PACKAGE.xml 
+             ├── AD_PROCESS_PARA.xml 
+             ├── AD_PROCESS.xml 
+             └── AD_TREENODE.xml            
+```
+
+- When defining the module inside the Application Dictionary, the only important considerations we must have are:
+
+    - Set the module type to *Module*
+
+    - Set the *Has Chart of Account* checkbox
+
+    - Remember to add the *dependency to Core*
+
+### Publishing the Chart of Accounts Module
+
+The way to package a chart of accounts module is similar to the [standard publishing process](.How_To_Create_and_Package_a_Module.md) for any module, just taking into account the previous particular considerations.
```

Comment:
The way to publish a chart of accounts module is similar to the [how to publish modules to github repository](http://docs.etendo.software/latest/developer-guide/etendo-classic/how-to-guides/how-to-publish-modules-to-github-repository/)  process.